keps/sig-node/0014-runtime-class.md

```diff
@@ -0,0 +1,359 @@
+---
+kep-number: 14
+title: Runtime Class
+authors:
+  - "@tallclair"
+owning-sig: sig-node
+participating-sigs:
+  - sig-architecture
+reviewers:
+  - TBD
+approvers:
+  - TBD
+editor: TBD
+creation-date: 2018-06-19
+status: provisional
+---
+
+# Runtime Class
+
+## Table of Contents
+
+* [Summary](#summary)
+* [Motivation](#motivation)
+  * [Goals](#goals)
+  * [Non\-Goals](#non-goals)
+  * [User Stories](#user-stories)
+* [Proposal](#proposal)
+  * [API](#api)
+    * [API Specification](#api-specification)
+    * [CRI API Specification](#cri-api-specification)
+  * [Versioning, Updates, and Rollouts](#versioning-updates-and-rollouts)
+  * [Implementation Details](#implementation-details)
+  * [Risks and Mitigations](#risks-and-mitigations)
+* [Graduation Criteria](#graduation-criteria)
+* [Implementation History](#implementation-history)
+* [Appendix](#appendix)
+  * [Examples of runtime variation](#examples-of-runtime-variation)
+
+## Summary
+
+`RuntimeClass` is a new cluster-scoped resource that surfaces container runtime properties to the
+control plane. RuntimeClasses are assigned to pods through a `runtimeClass` field on the
+`PodSpec`. This provides a new mechanism for supporting multiple runtimes in the cluster and on the
+nodes.
```

Comment:
nit: i think you mean we can support multiple runtimes on the same node at the same time. what would it mean to support multiple runtimes in the cluster ?

Reply:
clarified. It could be the case that you have some nodes that support one runtime and other nodes that support a different runtime (e.g. linux & windows), or a bunch of nodes that support 2 runtimes (e.g. native & sandboxed).